src/django_components/template_loader.py

```diff
@@ -37,6 +57,7 @@ def get_dirs(self) -> List[Path]:
 
         directories: Set[Path] = set()
         for component_dir in component_dirs:
+            # Consider tuples for STATICFILES_DIRS (See #489)
```

Comment:
TODO - Add link to https://docs.djangoproject.com/en/5.0/ref/settings/#prefixes-optional